ui/paladin/inputs.ts

```diff
@@ -22,6 +22,8 @@ export const StartingSealSelection = <SpecType extends PaladinSpecs>() =>
 		fieldName: 'seal',
 		values: [
 			{ actionId: ActionId.fromSpellId(31801), value: PaladinSeal.Truth },
+			{ actionId: ActionId.fromSpellId(20165), value: PaladinSeal.Insight },
+			{ actionId: ActionId.fromSpellId(20154), value: PaladinSeal.Righteousness },
```

Comment:
Adding these without #649 (which also adds them, and Justice, and all the actual implementations) won't do anything :) better to remove these two lines to avoid conflicts

Reply:
Yeah, I started the cleanup on the 8th so I didn't know anyone was working on it 😝  And a proper work at that, unlike my Wrath cleaning 😆 Removed to avoid unnecessary conflict 👍